docs/source/install/instructions/spack.rst

```diff
@@ -18,13 +18,13 @@ First `install spack <http://spack.readthedocs.io/en/latest/getting_started.html
 
    # get spack
    git clone https://github.com/spack/spack.git $HOME/src/spack
-   $HOME/src/spack/bin/spack bootstrap
 
    # activate the spack environment
-   # note: add this to your $HOME/.profile - otherwise you have to
-   #       do this every time you open a new terminal
    . $HOME/src/spack/share/spack/setup-env.sh
```

Comment:
offline discussed: change `. $HOME/...` to `source $HOME/...`

Reply:
thx, forgot that!